src/main/java/com/immunopass/entity/OrderEntity.java

```diff
@@ -31,18 +26,27 @@
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
+    @Column(name = "voucher_count")
```

Comment:
No need of @Column annotation as the mapping is handled by spring.jackson.property-naming-strategy=SNAKE_CASE property.

Reply:
Oh I see.